src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -511,6 +544,32 @@ def fuzz(self,
     fuzzer_process_result.command = self.fuzzer.last_fuzz_cmd
     return fuzzer_process_result
 
+  def merge(self,
+            corpus_directories,
+            merge_timeout,
+            artifact_prefix=None,
+            tmp_dir=None,
+            additional_args=None):
+    # TODO(flowerhack): Integrate some notion of a merge timeout.
+    # TODO(flowerhack): device.py doesn't have a notion of custom
+    # corpus directories, teach it
```

Comment:
This second TODO should get done & tested before actually merging this CL.

Merging "works" at the moment in that it doesn't *break* anything, but it seems like the names used for the temporary and merge directories on target and on the host differ, and thus CF fails to actually pull down the merged corpus D:

It shouldn't be hard to fix, just a bit fiddly, and it's late, so I'll wrap this bit up Monday morning.

The rest of the CL should be fine to look at / code review / etc, though

Reply:
fixed this. merge directories on host & target now sync and pull things down appropriately